src/global-shortcuts.c

```diff
@@ -299,6 +317,50 @@ shortcuts_bound_cb (GObject *source_object,
     }
 }
 
+static XdpOptionKey global_shortcuts_keys[] = {
+  { "description", G_VARIANT_TYPE_STRING, NULL },
+  { "preferred_trigger", G_VARIANT_TYPE_STRING, NULL },
+};
+
+gboolean
+xdp_verify_shortcuts (GVariant *shortcuts,
+                      GVariantBuilder *filtered,
+                      GError **error)
+{
+  gboolean ret = TRUE;
+  gchar *shortcut_name;
+  GVariant *values;
+
+  g_autoptr(GVariantIter) iter = g_variant_iter_new (shortcuts);
```

Comment:
A couple..

Reply:
Some `git grep` tells me that is to be the most common way in xdg-desktop-portal.

Reply:
This is the only other case where `g_variant_iter_new` is used:
https://github.com/flatpak/xdg-desktop-portal/blob/main/src/background.c#L264

Reply:
What I checked was `g_autoptr` in general.

Reply:
Sorry, done.